common/changes/@cadl-lang/rest/fix-empty-obj-204_2022-09-30-16-19.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@cadl-lang/rest",
+      "comment": "mpty object as a response will no",
```

Comment:
```suggestion
      "comment": "Empty object as a response will not produce a 204 response anymore",
```

Reply:
... seems like the copy and paste had some opinions